src/components/UploadingJobsDisplay/UploadingJobsDisplay.js

```diff
@@ -100,10 +107,10 @@ export class UploadingJobsDisplay extends Component {
   mapFilesToState() {
     const {
       history,
-      location: { state = {} },
+      location,
     } = this.props;
 
-    const { files } = state;
+    const { state: { files } = {} } = location;
```

Comment:
Since `get` util is already used in the file I think it makes sense to simplify this code.

`const files = get(location, 'state.files')`